code/game/objects/items/toys.dm

```diff
@@ -127,6 +127,64 @@
 	throw_range = 7
 	force = 0
 	var/random_color = TRUE
+	var/world_state
+	var/storage_state
+	var/current_color
```

Comment:
Also please add documenting comments to these vars (written above the var, prefixed with `///`, describe what the variable is for).